omod/src/main/webapp/pages/reports/interactiveRenderer.gsp

```diff
@@ -0,0 +1,257 @@
+<%
```

Comment:
There isn't really that much to this - a lot of the work in getting this to work was in improving the reportingrest module side.  We will want to iterate on this but this gives us a starting point.